### DISCUSSION TITLE
Entities not updating

### DISCUSSION THREAD

**Discussion by permolin** [klejejs/ha-thermia-heat-pump-integration#112] [Q&A]
I have recently installed the integration and the entities does not update unless i reload the integration. How do I make the integration refresh values automatically?

**Comment by klejejs** [comment]
The values should update automatically. Do you see any errors in the Home Assistant logs?
Also, it would help with debugging if you posted the data of the `thermia_debug.txt` file generated by the `thermia.debug` service.

**Reply by MrNickIE** [reply]
I too have just started having this problem, its an odd one!  Logs show success on the first reload

**Reply by flexi80** [reply]
I'm having the same issue, data stops updating. Starts again when the integration is reloaded. 
[thermia_debug.txt](https://github.com/user-attachments/files/18308119/thermia_debug.txt)

**Reply by permolin** [reply]
Sorry for the late answer. I have not been able to check this but will return shortly.

**Reply by permolin** [reply]
It seems that the integration started working on 2025-01-02 at 10:00. I can't recall that I did something with Home Assistant at that moment. But anyway, everything runs fine now!
